src/style.css

```diff
@@ -0,0 +1,18 @@
+@import url(./blocks/stars.css);
+@import url(./blocks/card.css);
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap");
+
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document